src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,71 @@
+package duke.task;
+
+/**
+ * Class for Task.
+ *
+ * @author Shawn Chew
+ * @version CS2103T AY 22/23 Sem 1
+ */
+public class Task {
+    /**
+     * The details of the task.
+     */
+    protected String description;
```

Comment:
It is perfectly fine to write Javadoc of class members on three lines, but it can also be specified on a single line as follows.
```suggestion
    /** The details of the task. */
    protected String description;
```